connect-examples/v2/csharp_checkout/Pages/Checkout.cshtml.cs

```diff
@@ -25,6 +25,7 @@ public CheckoutModel( Microsoft.Extensions.Configuration.IConfiguration configur
       client = new SquareClient.Builder()
         .Environment(environment)
         .AccessToken(configuration["AppSettings:AccessToken"])
+        .UserAgentDetail("sample_app_csharp_checkout")
```

Comment:
I think it might be helpful to add a comment to developers that they should remove this line when building on top of this sample. Something like
```cs
.UserAgentDetail("sample_app_csharp_checkout") // Remove or replace this detail when building your own app
```

Reply:
Same for all of the other samples as well.